tests/gui/TestGui.cpp

```diff
@@ -119,6 +120,43 @@ void TestGui::cleanup()
     m_dbWidget = nullptr;
 }
 
+void TestGui::testCreateDatabase()
+{
```

Comment:
Do we want to create this in the test data directory or in the temp directory? I don't think tests should be writing anything except to temp even if they do cleanup after themselves.

Reply:
Nice idea! Fixed